components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -704,14 +713,14 @@ - (CGFloat)headerAnimationDistance {
   CGFloat headerAnimationDistance =
       kHeaderAnimationDistanceAddedDistanceFromTopSafeAreaInset;
   if (self.contentReachesFullscreen) {
-    headerAnimationDistance += MDCDeviceTopSafeAreaInset();
+    headerAnimationDistance += [self topSafeArea];
   }
   return headerAnimationDistance;
 }
 
 - (CGFloat)addedContentHeightThreshold {
   // TODO: (#4900) change this to use safeAreaInsets as this is a soon to be deprecated API.
```

Comment:
_Nit:_ It looks like this TODO is addressed now, right?

Reply:
Done